src/Tizen.NUI.Wearable/src/public/CircularSlider.cs

```diff
@@ -0,0 +1,674 @@
+        });
+
+        /// <summary>Bindable property of ProgressColor</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty ProgressColorProperty = BindableProperty.Create("progressColor", typeof(Color), typeof(CircularSlider), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+
+            // TODO : Set viewStyle.ProgressColor after style refactoring done.
+
+            instance.UpdateProgressVisualColor((Color)newValue);
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            return ((CircularSliderStyle)((CircularSlider)bindable).viewStyle)?.ProgressColor;
+        });
+
+        /// <summary>Bindable property of ThumbSize</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty ThumbSizeProperty = BindableProperty.Create(nameof(ThumbSize), typeof(Size), typeof(CircularSlider), new Size(0,0), propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.thumbSize = (Size)newValue;
+                instance.UpdateThumbVisualSize((Size)newValue);
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.thumbSize;
+        });
+
+        /// <summary>Bindable property of ThumbColor</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty ThumbColorProperty = BindableProperty.Create("thumbColor", typeof(Color), typeof(CircularSlider), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+
+            // TODO : Set viewStyle.ThumbColor after style refactoring done.
+
+            instance.UpdateThumbVisualColor((Color)newValue);
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            return ((CircularSliderStyle)((CircularSlider)bindable).viewStyle)?.ThumbColor;
+        });
+
+        /// <summary>Bindable property of IsEnabled</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty IsEnabledProperty = BindableProperty.Create(nameof(IsEnabled), typeof(bool), typeof(CircularSlider), true, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.privateIsEnabled = (bool)newValue;
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.privateIsEnabled;
+        });
+
+        private static readonly string TrackVisualName = "Track";
```

Comment:
Use `const` instead of `static readonly` (StyleCop rule violation(CA1802)).
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1802?view=vs-2019